drivers/wifi/wfx200/wfx200_events.c

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2023 grandcentrix GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sl_wfx.h>
+#undef BIT
```

Comment:
-1, this is not allowed, fix `sl_wfx.h` instead.

Reply:
First, thanks for your review.
I wanted to keep sl_wfx.h as near as possible to the original. I'll fix that in the PR for the HAL

Reply:
I fixed that.